scripts/mbedtls_framework/c_parsing_helper.py

```diff
@@ -60,18 +62,37 @@ def __init__(self, #pylint: disable=too-many-arguments
                  qualifiers: Iterable[str],
                  return_type: str,
                  name: str,
-                 arguments: List[str]) -> None:
+                 arguments: List[str],
+                 doc: str = "") -> None:
+
         self.filename = filename
         self.line_number = line_number
         self.qualifiers = frozenset(qualifiers)
         self.return_type = return_type
         self.name = name
         self.arguments = [ArgumentInfo(arg) for arg in arguments]
+        self.doc = doc
 
     def returns_void(self) -> bool:
         """Whether the function returns void."""
         return bool(self.VOID_RE.search(self.return_type))
 
+    def __str__(self) -> str:
+        _str_args = [str(a) for a in self.arguments]
+        _str = "{} {} {}({})".format(" ".join(self.qualifiers),
+                                     self.return_type, self.name,
+                                     ", ".join(_str_args)).strip()
+        _str = FunctionInfo._c_align__(_str)
+        return self.doc + "\n" + _str
+
+    @staticmethod
+    def _c_align__(in_str: str) -> str:
```

Comment:
Please document this method. And rename it? It doesn't actually seem to do any aligning.